packages/ember-glimmer/lib/helpers/hash.js

```diff
@@ -0,0 +1,38 @@
+import { helper } from '../helper';
+/**
+@module ember
+@submodule ember-templates
+*/
+
+/**
+   Use the `{{hash}}` helper to create a hash to pass as an option to your
+   components. This is specially useful for contextual components where you can
+   just yield a hash:
+
+   ```handlebars
+   {{yield (hash
+      name='Sarah'
+      title=office
+   )}}
+   ```
```

Comment:
It's a bit ironic that we don't actually have a test for this case :kissing:

Can you quickly add a test for yielding a hash? I *think* it won't work for glimmer yet, so it probably needs to be `@htmlbars`'ed